public/css/style-narrower.css

```diff
@@ -17,7 +17,7 @@
 /* Section/Article */
 
 	header br {
-		display: none;
+/*    display: none;*/
```

Comment:
I'm happy to change this according to your design, but I commented it out to allow for the line breaks in the text to visually separate the content.